src/page/template/list/conversations.htm

```diff
@@ -20,7 +20,7 @@
   </div>
   <!-- ko using: callingViewModel -->
     <!-- ko foreach: {data: activeCalls, as: 'activeCall', noChildContext: true} -->
-      <!-- ko if: !activeCall.reason() -->
+      <!-- ko if: activeCall.reason() === CALL_REASON.NORMAL -->
```

Comment:
Since `activeCall.reason()` can be `undefined` and `0`, we should check what we actually want here.